### PR TITLE
Keychain and AppleDeviceFamily updates

### PR DESCRIPTION
**Pull request by olegoid**
Keychain:
* Add GetKeychainPath method

AppleDeviceFamily:
* Add iPod to the list device families
* Add `AppleDeviceFamilyExtensions.GetDeviceClass` it returnes string represnetation of `AppleDeviceFamily` enum members.

### CODE REVIEW

File: Xamarin.MacDev/IPhoneDeviceType.cs [hunk 1]
```diff
@@ -46,6 +46,22 @@ public enum AppleDeviceFamily
...
+	{
+		public static string GetDeviceClass (this AppleDeviceFamily family)
+		{
+			switch (family) {
+				case AppleDeviceFamily.TV: return "tvOS";
+				case AppleDeviceFamily.Watch: return "watch";
+				case AppleDeviceFamily.IPhone: return "iphone";
+				case AppleDeviceFamily.IPad: return "ipad";
+				case AppleDeviceFamily.IPod: return "ipod";
+				default: return string.Empty;
```
Comment (VincentDondain) [inline Xamarin.MacDev/IPhoneDeviceType.cs]: Similar to `ToDeviceType` you might want to throw on default.

`throw new ArgumentOutOfRangeException (string.Format ("Unknown device family: {0}", family));`

File: Xamarin.MacDev/IPhoneDeviceType.cs [hunk 2]
```diff
@@ -46,6 +46,22 @@ public enum AppleDeviceFamily
 		IPad   = 2,
 		TV     = 3,
 		Watch  = 4,
+		IPod   = 5,
+	}
+
+	public static class AppleDeviceFamilyExtensions
+	{
+		public static string GetDeviceClass (this AppleDeviceFamily family)
```
Comment (VincentDondain) [inline Xamarin.MacDev/IPhoneDeviceType.cs]: You could just name it `ToString` (:

File: Xamarin.MacDev/IPhoneDeviceType.cs [hunk 3]
```diff
@@ -46,6 +46,22 @@ public enum AppleDeviceFamily
...
+		{
+			switch (family) {
+				case AppleDeviceFamily.TV: return "tvOS";
```
Comment (VincentDondain) [inline Xamarin.MacDev/IPhoneDeviceType.cs]: Why "tvOS" and not "tv"? Are those strings user-facing? Cause "iphone" could be "iPhone" if that's the case.

**Comment by olegoid** [comment]
Thanks for the feedback @VincentDondain. I moved this extension method out of Xamarin.MacDev because it returns Apple Developer Portal specific strings and Xamarin.MacDev should know nothing about it.

**Pull request merged**